tico/utils/mx/mx_ops.py

```diff
@@ -0,0 +1,270 @@
+# Copyright (c) 2025 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This file was copied from https://github.com/microsoft/microxcaling/tree/v1.1.0
+# and modified for our purpose.
```

Comment:
This file was copied and modified. I just added `quantize_mx` function at the end of the file.